src/js/messaging/containers/ManageFolders.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+
+import {
+  deleteFolder,
+  openCreateFolderModal,
+} from '../actions';
+
+import ButtonCreateFolder from '../components/buttons/ButtonCreateFolder';
+import ButtonDelete from '../components/buttons/ButtonDelete';
+import { folderUrl } from '../utils/helpers';
+
+export class ManageFolders extends React.Component {
```

Comment:
This new file just splits the original Settings.jsx into the "Manage folders" tab content.